src/v1/controllers/post.controller.js

```diff
@@ -42,4 +42,22 @@ postRouter.get('/:id', async (request, response) => {
   response.json(readPost);
 });
 
+postRouter.patch('/:id', async (request, response) => {
+  const {
+    params: {
+      id,
+    },
+    body: {
+      // ????
```

Comment:
The properties pulled from the body should mirror the post controller/service `create` method. We're updating a `Post` model, so take a look at the [Post model](https://github.com/gmd298/personal-cms-api/blob/master/src/v1/models/post.model.js) (and compare it to the existing Post `create` method. to see what properties a user should be able to update. Take a look at the admin model vs the create + update methods in the admin controller/service.